### PR TITLE
Create 2021-69.md

### PR DESCRIPTION
**Pull request by billfox1**
Sprint 69 Release Notes

Erkang: I've rec'd Adam and Carter's notes, not sure if Phil has any, but here is the initial PR...

### CODE REVIEW

**Review by aiwilliams — DISMISSED** [review]
Seems good to me. 👍

File: release-notes/2021-69.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  a significant amount of data
+  
+### Jamf
+
+- Fixed a regression that prevented connections to Jamf servers running on port
+  `8443`.
+  
+### Qualys
+- Added additional properties to `discovered_host` `Host` mapped entities:
+  - `tags`: Simple Qualys asset tag values (i.e. `tags: ["Cloud Agent"]`)
```
Comment (erkangz) [inline release-notes/2021-69.md:78]: same here. space/empty line before and after. please fix other places too

File: release-notes/2021-69.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  | --------------------- | ------------------------------------------------------------- |
+  | API Gateway           | `google_api_gateway_api`                                      |
+  |                       | `google_api_gateway_api_config`                               |
+  |                       | `google_api_gateway_gateway`                                  |
+  | Certificate Authority | `google_privateca_certificate_authority`                      |
+  |                       | `google_privateca_certificate`                                |
+  | Cloud Functions       | `google_cloud_function` **USES** `google_iam_service_account` |
+  | Spanner               | `google_spanner_instance`                                     |
+  |                       | `google_spanner_instance_database`                            |
+  |                       | `google_spanner_instance_config`                              |
```
Comment (erkangz) [inline release-notes/2021-69.md:67]: space empty line needed

File: release-notes/2021-69.md [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  for details on what's currently supported.
+  
+### Gitlab
+
+- Fixed failure to complete ingestion for some large Gitlab accounts by moving
+  to latest infrastructure.
+  
+### Google Cloud
+
+- Added support for ingesting the following **new** resources:
```
Comment (erkangz) [inline release-notes/2021-69.md:56]: Space/empty line after

File: release-notes/2021-69.md [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+- Improved error messaging when `Directory.Read.All` permission is not present
+  and `config.ingestActiveDirectory=true`.
+- Fixed `DuplicateKeyError`s errors on `azure_vm_uses_storage_account` when
+  multiple data disks point to the same storage account.
+  
+### Cobalt
+
+Initial release of Cobalt.io integration (beta)! 🎉
+- Check out the
+  [docs](https://github.com/JupiterOne/graph-cobalt/blob/master/docs/jupiterone.md)
```
Comment (erkangz) [inline release-notes/2021-69.md:46]: we need to copy and publish the Cobalt integration doc into the docs repo, and use relative link here:

```
[docs](../docs/integrations/cobalt/index.md)
```

File: release-notes/2021-69.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+## Integrations
+
+### Azure
+
+- Added an option to auto-configure additional integrations for each
+  subscription in a directory/tenant that does not have a "JupiterOne" tag set
+  to "SKIP". Find this setting on the integration instance that you have
+  configured to ingest Active Directory data from Azure.
+- Added the `enabled` property to `azure_security_center_setting` entities.
+- Added support for ingesting the following **new** resources:
```
Comment (erkangz) [inline release-notes/2021-69.md:24]: space empty line needed before and after the table

**Review by erkangz — CHANGES_REQUESTED** [review]

File: release-notes/2021-69.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+
+## New Features and Improvements
+
+- Made the "Add new dashboard" to Insights view more clear by removing checkboxes (the view is single select, not multi select)
+
+- Made the alerts app sort by severity when it is loaded
+
+- Added a minimum height to compliance evidence query results
+
+- Updated all existing procedures to have considerations if there are considerations matching the templates found in GitHub
```
Comment (erkangz) [inline release-notes/2021-69.md:13]:
```suggestion
- Display "considerations" when available, in the editing view of a procedure in the Policies app. 
  Also updated all existing procedures to have considerations if available from the templates.
```

File: release-notes/2021-69.md (hunk 4 above)
Comment (billfox1) [reply release-notes/2021-69.md:47]: Replace line 46 link with the ../docs/integrations/cobalt/index.md?  Would you ping me the entire url when you get a chance?

**Review by erkangz — APPROVED** [review]

**Pull request merged**